pages/common/poetry-init.md

```diff
@@ -0,0 +1,28 @@
+# poetry init
+
+> Create a basic `pyproject.toml` file interactively.
```

Comment:
```suggestion
> Interactively create a `pyproject.toml` file in the current directory.
```

Reply:
Please make the description a bit more descriptive to clarify what the command does.